packages/scorchwrap/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "src/plugin.js",
   "scripts": {
-    "test": "ava --timeout=30s test/*.js",
+    "test": "ava --timeout=300s test/*.spec.js",
```

Comment:
this may be unnecessary - without the real dependency tree these tests are quite fast.

Reply:
300 is a lot